deploy/gateway/templates/podmonitor.yaml

```diff
@@ -14,7 +14,9 @@ spec:
       {{- include "gateway.selectorLabels" . | nindent 6 }}
   podMetricsEndpoints:
     - port: metrics
+      {{- if .Values.metrics.podMonitor.interval }}
```

Comment:
When `metrics.podMonitor.interval` is not provided, we shouldn't render an empty `interval` field in the template